src/test/java/uk/gov/digital/ho/pttg/api/SmokeTestsResultTests.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.digital.ho.pttg.api;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.PathNotFoundException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SmokeTestsResultTests {
+
+    private static final boolean ANY_BOOLEAN = false;
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    @Rule
```

Comment:
Could use 'assertThatThrownBy' or equivalent, instead of the @Rule

Reply:
Oh yeah. I used to always use that so I don't know why I changed!